basic.js

```diff
@@ -59,6 +61,26 @@ startButton.addEventListener(
 controls.addEventListener('lock', () => (menuPanel.style.display = 'none'))
 controls.addEventListener('unlock', () => (menuPanel.style.display = 'block'))
 
+const MAP_SIZE = 21;
+var greed = getMaze(MAP_SIZE);
+
+const wallMaterial = new THREE.MeshStandardMaterial( {color: 0xFFFFFF} ); //0x3C96FA
+const wallHeight = 10;
+for (var i=0; i<MAP_SIZE; i++){
+    for (var j=0; j<MAP_SIZE; j++) {
+        if (greed[i][j] == 1) {
+            console.log("construct wall ...")
+            const wallGeometry = new THREE.BoxGeometry(5, wallHeight, 5);
```

Comment:
This can reduce duplicated declaration of `wallGeometry`.
```suggestion
const wallGeometry = new THREE.BoxGeometry(5, wallHeight, 5);
for (var i=0; i<MAP_SIZE; i++){
    for (var j=0; j<MAP_SIZE; j++) {
        if (greed[i][j] == 1) {
            console.log("construct wall ...")
```